firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java

```diff
@@ -332,7 +332,7 @@ public NetworkRequestMetric build() {
     }
 
     if (isManualNetworkRequestMetric) {
-      logger.info(
+      logger.debug(
```

Comment:
Should this be warn?

Reply:
I think this could be `debug`. My interpretation of tis logging message is that manually created network metric cannot be built with `NetworkRequestMetricBuilder`. This is a check statement to inform that a manually created network metric is passed in the function. Therefore, a `debug` message could be sufficient.